src/main/scala/org/hammerlab/guacamole/loci/set/TakeLociIterator.scala

```diff
@@ -112,16 +119,16 @@ class TakeLociIterator(it: BufferedIterator[(Position, Coverage)],
     // If any Intervals have been found, build them into a [[Contig]] and return it, as well as the number of covering
     // regions.
     def result: Option[(Int, Contig)] =
-    if (intervals.isEmpty)
-      None
-    else
-      Some(
-        curNumRegions ->
-          Contig(
-            it.contigName,
-            intervals
-          )
-      )
+      if (intervals.isEmpty)
+        None
+      else
+        Some(
+          curNumRegions ->
+            Contig(
+              it.contigName,
+              intervals
+            )
+        )
```

Comment:
yea this `if`/`else` was not indented from the `def` it implements before, and it should be